src/nix/build.md

```diff
@@ -40,6 +40,18 @@ R""(
   lrwxrwxrwx 1 … ./result-dev -> /nix/store/dkm3gwl0xrx0wrw6zi5x3px3lpgjhlw4-glibc-2.32-dev
   ```
 
+* Build all outputs:
+
+  ```console
+  # nix build nixpkgs#openssl^* --print-out-paths
```

Comment:
```suggestion
  # nix build "nixpkgs#openssl^*" --print-out-paths
```

I'm afraid that most shells will try to expand the `*` otherwise (at least my zsh does)

Reply:
Right, good point

Reply:
Should it be single quotes? concerned about glob expansion

Reply:
I think double quotes will be enough in any reasonable shell (oxymoron, I know). At the worst, it shows the idea.